p2p/raintree/addrbook_utils_test.go

```diff
@@ -103,9 +109,17 @@ func BenchmarkAddrBookUpdates(b *testing.B) {
 			err = network.processAddrBookUpdates()
 			require.NoError(b, err)
 
-			require.Equal(b, len(network.addrList), n)
-			require.Equal(b, len(network.addrBookMap), n)
-			require.Equal(b, int(network.maxNumLevels), testCase.numExpectedLevels)
+			require.Equal(b, n, len(network.addrList), n)
```

Comment:
Is the last `, n` parameter needed?

Reply:
Nope, typo

Addressed in https://github.com/pokt-network/pocket/pull/243/commits/e643750335add3120dda77e586cc00ea17e58956